cloud-info/ams-wrapper.sh

```diff
@@ -60,28 +58,28 @@ if test "$CHECKIN_SECRETS_PATH" = ""; then
 		--middleware "$CLOUD_INFO_MIDDLEWARE" \
 		--ignore-share-errors \
 		--format glue21 >cloud-info.out
-elif test "$USE_ACCESS_TOKEN" -eq 1; then
-	# Case 2: access token style
-	cloud-info-provider-service --yaml-file "$CLOUD_INFO_CONFIG" \
-		--middleware "$CLOUD_INFO_MIDDLEWARE" \
-		--ignore-share-errors \
-		--auth-refresher accesstoken \
-		--format glue21 >cloud-info.out
 else
-	# Let's use the service account directly on the info provider
-	CHECKIN_DISCOVERY="https://aai.egi.eu/auth/realms/egi/.well-known/openid-configuration"
-	CLIENT_ID="$(yq -r '.fedcloudops.client_id' <"$CHECKIN_SECRETS_FILE")"
-	CLIENT_SECRET="$(yq -r '.fedcloudops.client_secret' <"$CHECKIN_SECRETS_FILE")"
+	# use service account for everyone
+	export OS_DISCOVERY_ENDPOINT="https://aai.egi.eu/auth/realms/egi/.well-known/openid-configuration"
```

Comment:
suggest passing these as environment variables to the container instead of hardcoded.

Reply:
ok, need to rethink a bit how the container is called. 
However some of the ones below are actually configuration for cloud-info-provider-service call so they need to stay

Reply:
Ok, let's put it on the backlog then.

Reply:
Added this one also: https://github.com/EGI-Federation/ansible-role-fedcloud-ops/issues/37